tests/test_channel_metrics/test_completely_bounded_trace_norm.py

```diff
@@ -12,6 +13,7 @@ def test_cb_trace_norm_quantum_channel():
     np.testing.assert_equal(completely_bounded_trace_norm(phi), 1)
 
 
+@pytest.mark.filterwarnings("ignore::DeprecationWarning")
```

Comment:
I'm wondering, do we want to ignore these? While these might be annoying to see in the output, having them is probably a sign that we might want to try to upgrade something?

Reply:
Yeah, latest versions of the packages not being installed in the virtual environment were causing these warnings. 

We don't want to ignore these. I figured out the issue with older packages being installed after this commit.